examples/demo.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-import {WarpedTime, TimeControls} from '../node/main.js'
+import {WarpedTime, Tick, TimeControls} from '../node/main.js'
 
+console.log(Tick)
 window.time = new WarpedTime()
+window.tick = new Tick()
```

Comment:
Remove this, you added it as a default right?

Reply:
yeah but having access to it on window lets you pause in the command line